elide-core/src/test/java/com/yahoo/elide/core/PaginationLogicTest.java

```diff
@@ -14,7 +14,7 @@
 import com.yahoo.elide.ElideSettingsBuilder;
 import com.yahoo.elide.annotation.Paginate;
 import com.yahoo.elide.core.exceptions.InvalidValueException;
-import com.yahoo.elide.core.pagination.Pagination;
+import com.yahoo.elide.core.pagination.PaginationImpl;
```

Comment:
Should we rename the file to `PaginationImplTest.java`

Reply:
👍